opengapps-files.mk

```diff
@@ -1,66 +1,7 @@
-GAPPS_SOURCES_ALL_PATH := $(GAPPS_SOURCES_PATH)/all
-
 # Always needed
-PRODUCT_COPY_FILES += $(GAPPS_SOURCES_ALL_PATH)/etc/permissions/com.google.widevine.software.drm.xml:system/etc/permissions/com.google.widevine.software.drm.xml \
-                      $(GAPPS_SOURCES_ALL_PATH)/etc/permissions/com.google.android.maps.xml:system/etc/permissions/com.google.android.maps.xml \
-                      $(GAPPS_SOURCES_ALL_PATH)/etc/permissions/com.google.android.media.effects.xml:system/etc/permissions/com.google.android.media.effects.xml \
-                      $(GAPPS_SOURCES_ALL_PATH)/etc/preferred-apps/google.xml:system/etc/preferred-apps/google.xml \
-                      $(GAPPS_SOURCES_ALL_PATH)/framework/com.google.widevine.software.drm.jar:system/framework/com.google.widevine.software.drm.jar \
-                      $(GAPPS_SOURCES_ALL_PATH)/framework/com.google.android.maps.jar:system/framework/com.google.android.maps.jar \
-                      $(GAPPS_SOURCES_ALL_PATH)/framework/com.google.android.dialer.support.jar:system/framework/com.google.android.dialer.support.jar \
-                      $(GAPPS_SOURCES_ALL_PATH)/framework/com.google.android.media.effects.jar:system/framework/com.google.android.media.effects.jar
+PRODUCT_COPY_FILES += $(call gapps-copy-to-system, all, etc/permissions etc/preffered-apps framework)
```

Comment:
Spelling error, should be `preferred-apps`.

@mfonville should we add `sysconfig/google.xml` to MM+? (always) and `sysconfig/google_build.xml` to MM+ for `stock` builds?